geopackage_validator/utils.py

```diff
@@ -50,3 +53,12 @@ def check_gdal_installed():
         sys.exit(
             "ERROR: cannot find GDAL/OGR modules, follow the instructions in the README to install these."
         )
+
+
+def load_config(file_path):
+    path = Path(file_path)
+    assert path.exists()
+    with path.open() as file_handler:
+        if path.suffix in (".yaml", ".yml"):
+            return yaml.load(file_handler, Loader=yaml.SafeLoader)
```

Comment:
yaml is toch een superset van json? dus kun je niet altijd yaml.load gebruiken?

Reply:
De self service gebruikt json. (zie ook https://github.com/PDOK/geopackage-validator/pull/63#discussion_r592963254 )

Reply:
ik bedoel voor het laden van een bestand. self service kan nog steeds json input gebruiken, want ik zou verwachten dat `yaml.load('{"foo": "bar"}') == json.load('{"foo": "bar"}')`. omdat json valide yaml is. m.a.w. je hebt de json.load niet nodig. maar nevermind, want dit is prima zo.